engines/director/detection_tables.h

```diff
@@ -7969,6 +7986,8 @@ static const DirectorGameDescription gameDescriptions[] = {
 	// Launcher included in european version of the game, contains instructions, lore and credits
 	WINGAME1("voyeur2", "Launcher", "voyeur2.exe", "e7bccfe54354d08a9d2d0e181f66406b", 2003432, 602),
 
+	WINGAME1_l("wettlauf", "", "wettlauf.exe", "d62438566e44826960fc16c5c23dbe43", 1523209, Common::DE_DEU, 650),
```

Comment:
We need a tail checksum here

Reply:
Done, changed all checksums for the new entries to tail checksums